### DISCUSSION TITLE
Welcome to no_clock Discussions!

### DISCUSSION THREAD

**Discussion by dcblack** [dcblack/no_clock#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

### Important

Please keep questions and discussions focused on `no_clock`. General questions on SystemC should be directed elsewhere.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DavidC-75** [comment]
Hi, it seems the latest implementation of no_clock is broken (doesn't even compile, using gcc 6.3.0).
I made what *I believe* are sensible fixes (see attached files)... but you probably want to review that ;-)
Hope it helps !
David

[no_clock.hpp.txt](https://github.com/dcblack/no_clock/files/7991163/no_clock.hpp.txt)
[no_clock_if.hpp.txt](https://github.com/dcblack/no_clock/files/7991164/no_clock_if.hpp.txt)
